config/management/operational/src/validator_state.rs

```diff
@@ -0,0 +1,72 @@
+// Copyright (c) The Diem Core Contributors
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::{json_rpc::JsonRpcClientWrapper, validator_config::DecryptedValidatorConfig};
+use diem_global_constants::{CONSENSUS_KEY, OWNER_ACCOUNT};
+use diem_management::error::Error;
+use serde::Serialize;
+use structopt::StructOpt;
+
+#[derive(Debug, Default, Serialize)]
+pub struct VerifyValidatorStateResult {
+    /// Check if local validator state is consistent with its on-chain.
+    /// True iff all subsequent checks are true.
+    //pub consistent: bool,
+
+    /// Check if a validator is in the latest validator set on-chain.
+    pub in_validator_set: Option<bool>,
```

Comment:
I don't feel `Option` is very necessary. But I think it may be useful to show if the check has been done (e.g., if it is `None`, that means this check has not been executed).